source/_data/MCPubs.bib

```diff
@@ -463,4 +456,14 @@ @InProceedings{leap:atc20
   title     = {Effectively Prefetching Remote Memory with {Leap}},
   year      = {2020},
   pages     = {843--857},
+
+  publist_confkey = {ATC'20},
+  publist_link = {paper || leap-atc20.pdf},
```

Comment:
File folder must match bib key

Reply:
eh, forgot to check this :(

Reply:
Oops, sorry I got this wrong due to the way GitHub folding the diff. The way it is currently is correct.